src/App.vue

```diff
@@ -1,28 +1,129 @@
 <template>
   <div id="app">
-    <img alt="Vue logo" src="./assets/logo.png">
-    <HelloWorld msg="Welcome to Your Vue.js App"/>
+    <!--Logo-->
+    <div class="centerRow">
+      <p class="upperP">П</p>
+      <p class="simpleO">о</p>
+      <p class="italicP">п</p>
+      <p class="upperO">О</p>
+      <p class="greenL">л</p>
+      <p class="pinkN">н</p>
+      <p class="latinR">R</p>
+      <p class="blackE">Е</p>
+      <p class="italic4">4</p>
+      <p class="yellowK">k</p>
+      <p class="brownA">A</p>
+    </div>
+    <!--Main content-->
+    <transition name="fade" mode="out-in">
+      <router-view/>
+    </transition>
+    <!--Spacers-->
+    <div></div>
+    <div></div>
+    <div></div>
+    <div></div>
+    <!--Disclaimer-->
+    <p class="padding">Warning! This is not a real mobile payment terminal. Do not try to transfer money to us! This
+      site is designed just to test Flutter web. Hope you enjoy it =)</p>
   </div>
 </template>
 
 <script>
-import HelloWorld from './components/HelloWorld.vue'
-
 export default {
   name: 'App',
-  components: {
-    HelloWorld
-  }
+  components: {}
 }
 </script>
 
 <style>
-#app {
-  font-family: Avenir, Helvetica, Arial, sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-  text-align: center;
-  color: #2c3e50;
-  margin-top: 60px;
-}
+  #app {
+    font-family: "Roboto Thin", sans-serif;
+    -webkit-font-smoothing: antialiased;
+    text-align: center;
+    color: #2c3e50;
+    background-color: #D2E9FC;
+    background-size: cover;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .centerRow {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: baseline;
+  }
+
+  .upperP {
```

Comment:
the class name reflects the style, later the design might change and the class name might confuse. 
note: usually CSS class names are kebab-cased but it is up to you what case to use in the project.
here we can avoid adding class names and use `:nth-child`